src/api/config/routes/api_routes.rb

```diff
@@ -278,6 +278,7 @@
       get 'staged_requests' => 'staged_requests#index', constraints: cons
       resource :staged_requests, only: [:create, :destroy], constraints: cons
     end
+    delete 'staging_projects/staged_requests/:request_id' => :destroy, constraints: cons, controller: 'staged_requests', as: 'single_staged_request'
```

Comment:
I think this new endpoint should be added to the API docs

Reply:
[nitpick] Since it is an api endpoint, and not used in the webui the ```as: 'single_staged_request``` shouldn't be necessary, right?